bitcoin-magazine/bitcoin-explained/accounts-for-bitcoin-easypaysy!.md

```diff
@@ -0,0 +1,1038 @@
+
+Sjors: 00:15:03
+
+Yeah, rabbit, big fish.
+
+Aaron: 00:15:04
+
+Whatever it was.
+That's much easier to remember.
+
+Sjors: 00:15:07
+
+Yeah that makes sense.
+
+Aaron: 00:15:08
+
+So then you can remember it and you can tell people and you know, you don't forget.
+It's much easier to remember.
+
+Sjors: 00:15:13
+
+And you can go on TV and say, you know, remember rabbit, big, fish.
+Then people can send money to rabbit, big, fish.
+And maybe that's slightly easier than giving them an address, though I'm a little skeptical whether that short name is such a big deal.
+But this is kind of where we get privacy and also the stealth address part.
+
+Aaron: 00:15:31
+
+Let's hear it.
+
+Sjors: 00:15:32
+
+So the first time you're using this as a recipient, so you're going to create an account, you're going to put a transaction on the blockchain.
+If then later on you receive coins, we're going to assume you use this third methodology, the nice privacy thing, it's going to arrive on different addresses.
+So that's very nice.
+But, if you're stupid, or if your wallet is stupid, it's going to combine coins that, you know, are related to that first transaction.
+The first transaction itself is not spendable because it's an OP_RETURN, but maybe it comes from another coin.
+
+Aaron: 00:16:04
+
+Sure.
+
+Sjors: 00:16:05
+
+And now you start combining coins and somebody can link your identity to everything you received.
+So you still need to be very wary of this cluster analysis risk.
+
+Aaron: 00:16:14
+
+Yeah, you need to...
+
+Sjors: 00:16:05
+
+A new wallet could avoid that and warn you about it.
+
+Aaron: 00:16:17
+
+Yeah, no, that's true.
+You have coins in your wallet which you're going to use to pay the fee.
+You probably have more in your wallet than just the fees, so you have change and it's then sort of linked to your account.
+And then after which the coins can be linked to even the coins you received privately.
+And that's something to be aware of, sure.
+
+Sjors: 00:16:40
+
+So I think it's this initial step where you have to announce something on the blockchain that creates this liability almost.
+Now for some parties that's actually kind of a feature or at least they can manage it.
+If you're an exchange, you know what you're doing, you're not gonna make those privacy mistakes.
+And one of the cool things is that you can, you can, you know, add a DNS record is one of the suggestions so that people can verify that, hey, this DNS record matches this big fish thing.
+And that's a nice way to double check.
+I could imagine hardware wallets could also embed some of these well-known codes so that if you're sending to Kraken or whatever, your hardware wallet could say, oh yeah that's actually an address that belongs to that exchange, at least the last time we we updated the firmware to know these accounts.
+
+Aaron: 00:17:28
+
+Mmm...
+
```

Comment:
Another example of a line that doesn't contribute to the transcript's content - you can just delete this and continue with the same speaker.